Python/codegen.c

```diff
@@ -675,6 +675,34 @@ codegen_leave_annotations_scope(compiler *c, location loc,
     ADDOP_I(c, loc, BUILD_MAP, annotations_len);
     ADDOP_IN_SCOPE(c, loc, RETURN_VALUE);
     PyCodeObject *co = _PyCompile_OptimizeAndAssemble(c, 1);
+
+    // We want the parameter to __annotate__ to be named "format" in the
+    // signature  shown by inspect.signature(), but we need to use a
+    // different name (.format) in the symtable so that if the name
+    // "format" appears in  the annotations, it does not get clobbered
```

Comment:
Extremely minor fiddling with the text of the comment here.

Please consolidate down to one space between "in" and "the".

Also, please consider changing "does not" to "doesn't"; "does not" sounds stiff.